### PR TITLE
Adds ipFowarding to ONVK docs

### PR DESCRIPTION
**Pull request by stevsmit**
Version(s):
4.14.

Issue:
https://issues.redhat.com/browse/OCPBUGS-22211

Link to docs preview:
https://66668--docspreview.netlify.app/openshift-enterprise/latest/networking/cluster-network-operator#nw-operator-cr-cno-object_cluster-network-operator

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nw-operator-cr.adoc [hunk 1]
```diff
@@ -284,6 +284,11 @@ endif::operator[]
  While migrating egress traffic, you can expect some disruption to workloads and service traffic until the Cluster Network Operator (CNO) successfully rolls out the changes.
 ====
 
+
+|`gatewayConfig.ipForwarding`
```
Comment (trozet) [inline modules/nw-operator-cr.adoc:288]: shouldn't this be just "ipForwarding" and down in the section where gatewayConfig is (line 338)? Like where routingViaHost is?

File: modules/nw-operator-cr.adoc [hunk 2]
```diff
@@ -284,6 +284,11 @@ endif::operator[]
...
+|`object`
```
Comment (trozet) [inline modules/nw-operator-cr.adoc:289]: Maybe rephrase:
Specify `Restricted` to forward all traffic related to OVN-Kubernetes only

to

Specify `Restricted` to only allow IP forwarding for Kubernetes related traffic.

**Review by trozet — CHANGES_REQUESTED** [review]

File: modules/nw-operator-cr.adoc (hunk 2 above)
Comment (stevsmit) [reply modules/nw-operator-cr.adoc]: @trozet Fixed.

File: modules/nw-operator-cr.adoc (hunk 1 above)
Comment (stevsmit) [reply modules/nw-operator-cr.adoc]: @trozet Moved. Thanks a lot!

File: modules/nw-operator-cr.adoc
Comment (DonaghBr) [inline modules/nw-operator-cr.adoc:1]: LGTM

File: modules/nw-operator-cr.adoc [hunk 3]
```diff
@@ -346,6 +346,10 @@ The default value is `false`.
 This field has an interaction with the Open vSwitch hardware offloading feature.
 If you set this field to `true`, you do not receive the performance benefits of the offloading because egress traffic is processed by the host networking stack.
 
+|`ipForwarding`
+|`object`
+|You can control IP forwarding for all traffic on OVN-Kubernetes managed interfaces by using the `ipForwarding` specification in the `Network` resource. Specify `Restricted` to only allow IP forwarding for Kubernetes related traffic. Specify `Global` to allow forwarding of all IP traffic. For new installations, the default is `Restricted`. For upgrades to {product-title} 4.14, the default is `Global`.
```
Comment (skopacz1) [inline modules/nw-operator-cr.adoc:351]: [Update is the preferred term](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#update) over upgrade:

```suggestion
|You can control IP forwarding for all traffic on OVN-Kubernetes managed interfaces by using the `ipForwarding` specification in the `Network` resource. Specify `Restricted` to only allow IP forwarding for Kubernetes related traffic. Specify `Global` to allow forwarding of all IP traffic. For new installations, the default is `Restricted`. For updates to {product-title} 4.14, the default is `Global`.
```

**Review by skopacz1 — COMMENTED** [review]
Left one nit, otherwise looks good to me!

**Comment by stevsmit** [comment]
cherry-pick enterprise-4.14
